aptos/contracts/sources/price_status.move

```diff
@@ -0,0 +1,35 @@
+module pyth::price_status {
+    
+    /// The price feed is not currently updating for an unknown reason.
+    const UNKNOWN: u64 = 0;
+    /// The price feed is updating as expected.
+    const TRADING: u64 = 1;
+
+    /// PriceStatus represents the availability status of a price feed.
+    /// Prices should only be used if they have a status of trading.
+    struct PriceStatus has copy, drop, store {
+        status: u64,
+    }
+
+    public fun from_u64(status: u64): PriceStatus {
+        PriceStatus {
```

Comment:
nitpick: I don't know how enum is implemented here nor whether it's important at all. I think here checking `status < 2` could be good.

Reply:
Unfortunately Move doesn't have native enums. You're right though I'll definitely add that check.